utils_cv/tracking/model.py

```diff
@@ -177,15 +178,51 @@ def fit(
         trainer.set_device(
             self.opt.gpus, self.opt.chunk_sizes, self.opt.device
         )
-
+        
+        # initialize loss vars
+        self.losses_dict = defaultdict(list)
+        self.epochs = []
+        
         # training loop
         for epoch in range(self.start_epoch + 1, self.opt.num_epochs + 1):
+            print("Training in epoch ", epoch,"...")
             mark = epoch if self.opt.save_all else "last"
             log_dict_train, _ = trainer.train(epoch, train_loader)
             if epoch in self.opt.lr_step:
                 lr = self.opt.lr * (0.1 ** (self.opt.lr_step.index(epoch) + 1))
                 for param_group in optimizer.param_groups:
                     param_group["lr"] = lr
+                    
+            # store losses in each epoch            
+            self.epochs.append(epoch)
```

Comment:
Is this self.epochs used anywhere (except in the plot function where it might not be needed)